core/domain/topic_domain.py

```diff
@@ -337,10 +337,9 @@ def validate(self) -> None:
             ValidationError. One or more attributes of the StoryReference are
                 invalid.
         """
-        if not isinstance(self.story_is_published, bool):
+        if self.story_id == '':
```

Comment:
Why is this check added?

Reply:
After removing type checks, this `validate` method was empty, then I picked this check randomly. Do you want me to change this check?

Reply:
Hmm, I think it is fine, can we also add some length limit or regex check?

Reply:
yup, we can. I added a length check, please have a look and let me know if you want me to add anything else.

Reply:
Let just keep the regex check

Reply:
Yes, I did. Please have a look and let me know is it ok or not.